### PR TITLE
feat(i18n): add Portuguese translations

### PR DESCRIPTION
**Pull request by Makley-Tibola-Trichez**
Add Portuguese for @valibot/i18n

### CODE REVIEW

**Comment by fabian-hiller** [comment]
Thank you for this PR! Do you know another native speaking developer that could review the translations for me?

**Comment by Makley-Tibola-Trichez** [comment]
Yes, i'll ask someone to review and comment

File: packages/i18n/src/pt.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    minValue:       (issue) => `Valor inválido: Esperado ${issue.expected}, mas recebido ${issue.received}`,
+    multipleOf:     (issue) => `Múltiplo inválido: Esperado ${issue.expected}, mas recebido ${issue.received}`,
+    notBytes:       (issue) => `Bytes inválidos: Esperado ${issue.expected}, mas recebido ${issue.received}`,
+    notLength:      (issue) => `Tamanho inválido: Esperado ${issue.expected}, mas recebido ${issue.received}`,
+    notSize:        (issue) => `Tamanho inválido: Esperado ${issue.expected}, mas recebido ${issue.received}`,
+    notValue:       (issue) => `Valor inválido: Esperado ${issue.expected}, mas recebido ${issue.received}`,
+    octal:          (issue) => `Octal inválido: Recebido ${issue.received}`,
+    regex:          (issue) => `Formato inválido: Esperado ${issue.expected}, mas recebido ${issue.received}`,
+    safeInteger:    (issue) => `Inteiro seguro inválido: Recebido ${issue.received}`,
+    size:           (issue) => `Tamanho inválido: Esperado ${issue.expected}, mas recebido ${issue.received}`,
```
Comment (patrickpiccini) [inline packages/i18n/src/pt.ts:57]: `Tamanho inválido: Esperado ${issue.expected}, porém foi recebido ${issue.received}`

File: packages/i18n/src/pt.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    startsWith:     (issue) => `Início inválido: Esperado ${issue.expected}, mas recebido ${issue.received}`,
```
Comment (patrickpiccini) [inline packages/i18n/src/pt.ts:58]: `Início inválido: Esperado ${issue.expected}, porém foi recebido ${issue.received}`

File: packages/i18n/src/pt.ts [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    ulid:           (issue) => `ULID inválido: Recebido ${issue.received}`,
+    url:            (issue) => `URL inválido: Recebido ${issue.received}`,
+    uuid:           (issue) => `UUID inválido: Recebido ${issue.received}`,
+    value:          (issue) => `Valor inválido: Esperado ${issue.expected}, mas recebido ${issue.received}`,
```
Comment (patrickpiccini) [inline packages/i18n/src/pt.ts:62]: `Valor inválido: Esperado ${issue.expected}, porém foi recebido ${issue.received}`

**Review by patrickpiccini — CHANGES_REQUESTED** [review]

**Comment by Makley-Tibola-Trichez** [comment]
I believe everything is fine, but another friend will review it later

**Comment by fabian-hiller** [comment]
Thank you so much!

**Review by darc591 — APPROVED** [review]
LGTM 👍

**Comment by fabian-hiller** [comment]
Thank you everyone for this contribution!

**Pull request merged**